blog/2018/12/07/openapi/index.markdown

```diff
@@ -0,0 +1,169 @@
+# MetaCPAN, Mojolicious and OpenAPI
+
+During this years [meta::hack 3](http://www.olafalders.com/2018/11/21/metahack-3-wrap-report/), I was extremely fortunate to work with Joel Berger on integrating/documenting OpenAPI with the MetaCPAN API via Mojolicious.
+
+## What is it?
+OpenAPI is a specification for designing, documenting, validating and driving your RESTful API. It can be used to provide documentation to an existing API, or staring fresh.
+
+The OpenAPI Specification originated as the Swagger specification and was renamed to separate the API description format from the open source tooling. The specification moved into new GitHub repository, but did not change.
+
+In the case of the MetaCPAN API, we set out to provide documentation to the existing API, but quickly moved into supporting validation to API calls as well.
+
+---
+
+## The tools
+There are many tools available for working with OpenAPI, including discovery tools that will assist in writing your OpenAPI specification. Our choice was to write the definition by hand (in vim of course) and use tools to generate the documentation, and integrate the specification into MetaCPAN.
```

Comment:
Passive voice should be avoided. 

"OpenAPI has many tools available to help, including discovery tools that will assist in writing the specification. We chose to write the definition by hand (in vim of course) and use tools to generate the documentation and to integrate the specification into MetaCPAN."